config/defaults.go

```diff
@@ -13,5 +13,6 @@ const (
 	DefaultStoragePort           = "8525"
 	DefaultPublicAPIProxyPort    = "8535"
 	DefaultJSONRPCProxyPort      = "8545"
+	DefaultBotHealthCheckPort    = "8080"
```

Comment:
should we go with a less mainstream port? sometimes it makes things hard to debug

Reply:
Got any suggestions? Maybe I could do 8081?